en/developer_guide/navigation_panel.rst

```diff
@@ -41,14 +41,53 @@ The menu link must have the following attributes :
 * id : merge of the "nav-" string and the name of the strategy as returned by
   ``NavigationPanelInterface::getName()``, for instance "nav-user"
 
-It can also have data-* attributes that can help you to define custom properties to be used in a Backbone route,
-for example :
+Menu and data table
+~~~~~~~~~~~~~~~~~~~
+
+Many panel entries allow to consult records formated in a data table.
+This data table could be configured : name of the column, visible or not by default, type of search field...
+As this view is based on `DataTables library`_, all possible options in column definition are allowed.
```

Comment:
This sentence is very confusing. I can figure out what you're talking about, but only because I have the context. I think you should talk to someone who doesn't know the library, just telling him some configurations are possible and enumerate them.